src/index.html

```diff
@@ -4,19 +4,108 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo header logo"
+        />
```

Comment:
The `alt` attribute should describe the image content. It is good practice to be specific, but since this is a logo, it would be better to describe it as a 'company logo' for clarity.